### PR TITLE
Add GitHub ribbon to developer docs

### LINKED ISSUE #40: Add GitHub links on developer.workarea.com

### PR DESCRIPTION
**Pull request by meowsus**
Closes #40

No changelog

### CODE REVIEW

**Comment by meowsus** [comment]
Screenshot:

Screenshot from 2019-09-16 11-55-01

**Comment by tubbo** [comment]
I've never seen the ribbon down in the bottom right corner, interesting way to solve the problem of "workarea.com" being in the place where ribbons usually go

**Comment by bencrouse** [comment]
Nice! @meowsus can you add this to the homepage too to clean up the visuals on that resources section?
tmp

File: docs/source/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<% @hide_github_badge = true %>
```
Comment (bencrouse) [inline docs/source/index.html.erb]: (minor) Try not to make readers think in double negatives (e.g. `@hide_github_badge = false`). Easier to grok if it was `show_github_badge = true`).

**Pull request merged**